meta-spi/recipes-config/spi-config/spi-config_0.1.bb

```diff
@@ -0,0 +1,44 @@
+# See http://git.yoctoproject.org/cgit.cgi/poky/tree/meta/files/common-licenses
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+# TODO: Set this  with the path to your assignments rep.  Use ssh protocol and see lecture notes
+# about how to setup ssh-agent for passwordless access
+SRC_URI = "git://git@github.com/cu-ecen-aeld/Finalproject-Shreyan-Bala-shared.git;protocol=ssh;branch=master"
```

Comment:
@shreyan-collab can you put a README at https://github.com/cu-ecen-aeld/Finalproject-Shreyan-Bala-shared/tree/master/spi_package detailing where this utility came from (my guess is https://github.com/torvalds/linux/blob/master/tools/spi/spidev_test.c but it would also be great to link a specific version) and also what, if any changes you needed to make to it?